wordpress-mobile/WordPress-Android
# [Gutenberg] Use JS bundle from JitPack

This PR removes the dependency on Travis+AWS to get the Gutenberg JS bundle and instead, expects the file to be available via the bridge aar as an assets resource and resources merging.

To test:

1. Build and run the wpandroid app having the `wp.BUILD_GUTENBERG_FROM_SOURCE` gradle flag to true so the build will depend on the prebuilt JitPack library artifacts. The app should compile and run fine.
2. Run the app, enable GB in the App Settings and open a GB post. It should work as normal.
3. Check out the `<rootfolder>/WordPress/src/main/assets` folder and make sure there is no `index.android.bundle` file present. That used to be the JS bundle loaded by GB but now that lives inside the bridge library aar.

Update release notes:
